packages/ETLoad.package/ETLSegmentLoader.class/class/loadImageSegmentFrom.outPointers..st

```
@@ -17,4 +17,6 @@ loadImageSegmentFrom: segmentWordArray outPointers: outPointerArray
 		placeholder isLoadable ifFalse: [ ETLIncompatibleObjectError
 			signal: 'Loading classes is not yet supported'. ].
 		placeholder loadUsing: oopMap. ].
-	^ placeholders values first.
\ No newline at end of file
+	rootObject := placeholders values at: 1 ifAbsent: [ nil. ].
```

Again, would it not be better to fail instead of returning `nil`?